benchmarks/osb/params/no-train-params.json

```diff
@@ -10,6 +10,8 @@
   "target_index_bulk_index_data_set_format": "hdf5",
   "target_index_bulk_index_data_set_path": "<path to data>",
   "target_index_bulk_index_clients": 10,
+  "target_index_max_num_segments": 1,
```

Comment:
why we are making max segments to 1? what kind of benchmark we are expecting here?

Reply:
I think we could make this 10. Purpose of forcemerging is to get comparable. results during search phase. However, this will need to be adjusted depending on the data set being used and the insights to be gained.